moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h

```diff
@@ -186,8 +186,8 @@ private Q_SLOTS:
   void collisionObjectChanged(QListWidgetItem* item);
   void imProcessFeedback(visualization_msgs::InteractiveMarkerFeedback& feedback);
   void copySelectedCollisionObject();
-  void exportAsTextButtonClicked();
-  void importFromTextButtonClicked();
+  void exportGeometryAsTextButtonClicked();
```

Comment:
[export|import]**Scene**[As|From]Text...?

Reply:
The function it calls in PlanningScene is also called [saveGeometryToStream](https://github.com/ros-planning/moveit/blob/master/moveit_core/planning_scene/src/planning_scene.cpp#L952) and only saves the shapes in the world, ignoring subframes, object poses and even `AttachedBody` objects. I assume it is meant to load obstacle configurations. I added a tooltip to make this clear.

The existing `load_scene_button` seems to preserve the metadata, although I haven't tested it.